Exersize2/Exersize2/Program.cs

```diff
@@ -8,21 +8,24 @@ public static void Main(string[] args)
         {
 
 
-            for (int i = 1, i <= 50, i++)
+            for (int i = 1; i <= 50; i++)
             {
-                int a = i / 3;
-                int b = i / 5;
-                if ((Convert.ToDouble(i / 3) = i / 3) & (Convert.ToDouble(i / 5) != i / 5))
-                
+                double a =  i % 3.0;
+                double b = i % 5.0;
+                if ((a == 0) && (b != 0))
+                    
```

Comment:
delete empty string

Reply:
deleted. but is it obligatory? i like free space